app/uk/gov/hmrc/emcstfe/models/request/GetDraftMovementSearchOptions.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.emcstfe.models.request
+
+import play.api.mvc.QueryStringBindable
+import uk.gov.hmrc.emcstfe.models.common.{Descending, DestinationType, SortOrdering}
+import uk.gov.hmrc.emcstfe.models.request.GetDraftMovementSearchOptions.DEFAULT_START_POSITION
+
+import java.time.LocalDate
+
+case class GetDraftMovementSearchOptions(sortField: GetDraftMovementSortField = GetDraftMovementSearchOptions.DEFAULT_SORT_FIELD,
+                                         sortOrder: SortOrdering = GetDraftMovementSearchOptions.DEFAULT_SORT_ORDER,
+                                         startPosition: Int = DEFAULT_START_POSITION,
+                                         maxRows: Int = GetDraftMovementSearchOptions.DEFAULT_MAX_ROWS,
+                                         searchTerm: Option[String] = None,
```

Comment:
Is this going to be sanitised at FE level?

Reply:
Should the search term be decoded?

Reply:
It should be Escaped at the Frontend yeah; it would have to be to be able to be parsed as a URL? Whether we should do any further checks to block is an interesting one.